### ISSUE TITLE
Error ConnectionString in Helper.cs in PostgreSQL

### ISSUE THREAD

**Issue by VNA74** [aelassas/wexflow#18]
In ConnectionString not present database name **Database=" + databaseName**

`public void CreateDatabaseIfNotExists(string server, string userId, string password, string databaseName)
        {
            using (var conn = new NpgsqlConnection("Server=" + server + ";User Id=" + userId + ";Password=" + password))`

**Comment by aelassas** [comment]
Database field is not required in the connection string in Helper.cs because we use CREATE DATABASE command later on:

```
var command2 = new NpgsqlCommand("CREATE DATABASE " + databaseName + ";", conn)
```

**Comment by VNA74** [comment]
When run `conn.Open();` without **Database**  I get error. By default PostgreSQL try find database as User Id

**Comment by aelassas** [comment]
Fixed.

Check out this [commit](https://github.com/aelassas/wexflow/commit/c5245696e8e01039816a2c899fd34a690a3bf3b1).

**Comment by VNA74** [comment]
Thanks It works. You can also add a port to connect

**Comment by aelassas** [comment]
@VNA74 

I added the port.

Check out this [commit](https://github.com/aelassas/wexflow/commit/d9b4fb3fc573e7eb646ccb0742f80832ecd3c0f6).

**Comment by VNA74** [comment]
great, it works

**Comment by aelassas** [comment]
@VNA74 I released the fix in the [v6.6](https://github.com/aelassas/wexflow/releases/tag/v6.6).

**Comment by marijazvarych3** [comment]
> When run `conn.Open();` without **Database** I get error. By default PostgreSQL try find database as User Id
